### PR TITLE
chore(unit): enable unit testing

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

**Review by belom88 — CHANGES_REQUESTED** [review]
```
user@user-XPS-15-9570:~/apps/loaders.gl-showcases$ git clean -dfx
user@user-XPS-15-9570:~/apps/loaders.gl-showcases$ yarn
...
user@user-XPS-15-9570:~/apps/loaders.gl-showcases$ yarn test
yarn run v1.22.10
$ jest
/bin/sh: 1: jest: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6295 @@
+{
```
Comment (belom88) [inline package-lock.json]: As we use `yarn` remove `package-lock.json`

File: tsconfig.json [hunk 2]
```diff
@@ -13,5 +13,6 @@
     "sourceMap": true,
     // Docs - https://www.typescriptlang.org/tsconfig#noImplicitAny
     "noImplicitAny": false
-  }
+  },
+  "exclude": ["./src/**/*.spec.ts", "./src/**/*.spec.tsx", "node_modules"]
```
Comment (belom88) [inline tsconfig.json:17]: Code looks working even without this `exclude`

**Review by belom88 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (dryabinin-actionengine) [reply package-lock.json]: Removed.

File: tsconfig.json (hunk 2 above)
Comment (dryabinin-actionengine) [reply tsconfig.json]: Done

**Pull request merged**